src/examples/03_custom_mobjects/screen_grid.py

```diff
@@ -0,0 +1,99 @@
+from manimlib.imports import *
+
+
+class Grid(VMobject):
+    CONFIG = {
+        "height": 6.0,
+        "width": 6.0,
+    }
+
+    def __init__(self, rows, columns, **kwargs):
+        digest_config(self, kwargs, locals())
+        VMobject.__init__(self, **kwargs)
```

Comment:
I'm a bit confused about this digest_config stuff. I've only saw it on `Scene` classes until now, and there it get's done automatically by manim. Why do we need it here?

Reply:
`digest_config` it is used so that the parameters in the CONFIG dictionary can be modified later, in case this line is not added then the CONFIG parameters cannot be modified.